### PR TITLE
Use URIs instead of strings in some places

### PR DESCRIPTION
**Pull request by aelred**

### CODE REVIEW

**Comment by aelred** [comment]
!rtg - cass timeout

**Comment by aelred** [comment]
@fppt finally! A pass. Please tell me all the problems

File: grakn-engine/src/test/java/ai/grakn/client/TaskClientTest.java [hunk 1]
```diff
@@ -1,174 +0,0 @@
...
-import org.mockito.Mockito;
-
-import java.time.Duration;
-import java.time.Instant;
-
-import static ai.grakn.engine.TaskStatus.CREATED;
-import static java.time.Instant.now;
-
-// TODO: move this test into grakn-client when possible
-public class TaskClientTest {
```
Comment (fppt) [inline grakn-engine/src/test/java/ai/grakn/client/TaskClientTest.java]: Why is this test dieing?

Comment (aelred) [reply grakn-engine/src/test/java/ai/grakn/client/TaskClientTest.java]: because I merged this PR like 10 times with master 👍 
I'll try and resurrect it

**Comment by fppt** [comment]
screen shot 2017-11-09 at 11 12 37

I have no idea how that feels

**Comment by fppt** [comment]
Hahahahahahahaha one more merge @aelred

**Comment by fppt** [comment]
Okay in this case I am sorry. I merged another PR and this one conflicted :(

**Comment by aelred** [comment]
Finally!!!!

**Comment by fppt** [comment]
hahahaha I was about to merge :D

**Pull request merged**